libraries/AP_HAL_ChibiOS/hwdef/CubeOrangePlus-ODID/hwdef.dat

```diff
@@ -0,0 +1,105 @@
+# hw definition file for processing by chibios_hwdef.py
+
+# MCU class and specific type
+MCU STM32H7xx STM32H757xx
+
+define CORE_CM7
+define SMPS_PWR
+
+include ../CubeOrange/hwdef.inc
+
+undef USB_STRING_PRODUCT
+undef USB_STRING_MANUFACTURER
+undef APJ_BOARD_ID
+undef USB_PRODUCT
+
+# USB setup
+USB_PRODUCT 0x1058
+USB_STRING_MANUFACTURER "CubePilot"
+USB_STRING_PRODUCT "CubeOrange+"
+
+APJ_BOARD_ID 1063
```

Comment:
Shouldn't this be 11063?

Reply:
Thank you for the code review! Yes this should be 11063, I modified the commit to reflect this change.